src/collects/seashell/compiler/compiler.rkt

```diff
@@ -40,9 +40,10 @@
 
 ;; (seashell-resolve-dependencies file)
 ;; Invokes the internal preprocessor to resolve dependencies
-(: seashell-resolve-dependencies (-> Path (Values (Listof Path) (Listof Seashell-Diagnostic))))
-(define (seashell-resolve-dependencies file)
+(: seashell-resolve-dependencies (-> Path Path (Values (Listof Path) (Listof Seashell-Diagnostic))))
+(define (seashell-resolve-dependencies question-dir file)
```

Comment:
Maybe instead of question-dir pass in the clang flags that we pass to the main clang instance.